pkg/agent/secondarynetwork/podwatch/controller_test.go

```diff
@@ -285,29 +470,30 @@ func TestPodControllerAddPod(t *testing.T) {
 				InterfaceRequest: "eth11",
 			},
 		)
-		network1 := testNetwork("net1")
-		network2 := testNetwork("net2")
+		savedCNIConfig := *cniConfig
+		network1 := testNetwork("net1", sriovNetworkType)
+		network2 := testNetworkExt("net2", "", vlanNetworkType, defaultMTU, 100)
```

Comment:
nit: define a local constant for the vlan id

Reply:
Done